css/values.css

```diff
@@ -0,0 +1,557 @@
+/*
+===================================
+===================================
+Styles for an individual page
+
+- Rename this file to match the page you are working on (e.g. home.css, about.css, etc.)
+- Create a new file for your page and link this file to it in the <head> of your HTML document
+- Variables from global.css will work in this CSS file as well. Do not add any variables to this file.
+===================================
+===================================
+*/
+
+/* need to add wrapper */
+/* red background for bottom section needs fixing */
+/* some text changes from white to black when in desktop mode */
+/* blue background for support us needs fixing */
+/* heart color and gap needed for support us */
+/* white icons in support us need flexbox to move side by side in desktop mode */
+/* align title text to center somehow hoping wrapper fixes some stuff */
+html {
+	/* does this need before and after */
+	box-sizing: border-box;
+}
+
+/* 
+===================================
+Page Component Styles (icon grid, team member card)
+===================================
+*/
+.hero {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.hero img {
+	width: 100%;
+}
+.values-card {
+	min-height: 204px;
+	color: var(--color-white);
+	border: none;
+	border-radius: 10px;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	text-align: center;
+	margin: 0px 20px;
+}
+.values-card img {
+	margin-bottom: 20px;
+}
+.values-card h2 {
+	margin: 20px 0px 0px 0px;
+	font-size: var(--font28);
+	font-weight: var(--font-weight-bold);
+	line-height: var(--line-height32);
+}
+.values-card p {
+	display: flex;
+	text-align: left;
+	font-size: var(--font16);
+	font-weight: var(--font-weight-regular);
+	line-height: var(--line-height20);
+	margin: var(--spacing20);
```

Comment:
Make this `margin: var(--spacing20) 0;` to remove the margin from the left and right. This spacing will be done by the padding property that you add to `.values-card`.

Reply:
Done!